Dockerfile.ocp

```diff
@@ -1,9 +1,9 @@
 FROM registry.ci.openshift.org/ocp/builder:rhel-8-base-openshift-4.9
 
 RUN dnf upgrade -y \
- && dnf install -y qemu-img jq xz libguestfs-tools \
+ && dnf install -y qemu-img jq xz libguestfs-tools coreos-installer \
```

Comment:
This is no longer needed

Reply:
Actually we need this to modify the kargs when the IP_OPTIONS env is set.

Reply:
https://github.com/kirankt/ironic-rhcos-downloader/blob/8b6065832070c8d3ce76cbc040bf5894a8865900/get-live-images.sh#L64

Reply:
Ah, yep. I wonder if we can/should move this to ironic-image as well.